Automattic/jetpack
# Social: Tailor editor for the social note post type

This change updates the editor layout so we hide the yet unsupported parts of it.

## Proposed changes:

* Added a property to the publicize hook to check if the post type is a social note. dd982919a3b851126a7d263b21c8aab829e95167
* Hid the custom message box 4a68d6018a71fe3e2a4545a5ccca1387b03b9538
* Hid SIG and turned it off for the social notes 97cb48977d80ca474c3049c1c1b3b2d77053515b

### Other information:


## Jetpack product discussion


https://github.com/Automattic/jetpack-reach/issues/195
## Does this pull request change what data or activity we track or use?


No.
## Testing instructions:






* Check out this PR.
* Create a new social note, and check that in the editor you only see the attached media settings.
* Share a note with an attached media as a social post, and without the social post toggled, they should work as expected.

![CleanShot 2024-02-08 at 11 01 59 png](https://github.com/Automattic/jetpack/assets/36671565/7dac6b93-b985-4e05-b05c-5f20ef2f08ea)
